common/scripts/base.sh

```diff
@@ -99,8 +99,7 @@ function undeploy() {
 # $2 - service port
 function waitUntilServiceIsActive() {
   marathon_lb_host_ip=$(dcos marathon app show marathon-lb | $mesos_artifacts_home/common/scripts/get-marathon-lb-host.py)
```

Comment:
Since we get the IP of the marathon lb, the management console URL is displayed using the IP. Shall we use the lb hostname here?